domains/yunexz.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Yunexz",
+        "email": "literallynoname13@gmail.com"
+    },
+    "record": {
+        "A": ["45.11.229.211"]
```

Comment:
I cannot reach your website
> This site can’t be reached
> 45.11.229.211 took too long to respond.

Reply:
i just changed my email, does this affect me?